src/adapters/leasing-adapter.ts

```diff
@@ -477,6 +477,112 @@ const getOffersForContact = async (
   }
 }
 
+const validateRentalRules = (
+  validationResult: any,
```

Comment:
Tycker inte det ska vara `any` här. Det kan i princip komma in vad som helst här då och "löftet" om vad som returneras blir svårt att lita på.

Reply:
Det är ett AxiosResponse jag vill skicka in. Den här metoden är inte tänkt att exponeras utan bara för hjälp till validateResidentialAreaRentalRules och validatePropertyRentalRules som exporteras

Reply:
Uppdaterad 👍